glue/sales_val_flagging.py

```diff
@@ -427,19 +427,8 @@ def finish_flags(df, start_date, manual_update):
     )
     run_id = timestamp + "-" + random_words
 
-    # Control flow for incorporating versioning
-    dynamic_assignment = {
-        "run_id": run_id,
-        "rolling_window": lambda df: pd.to_datetime(
-            df["rolling_window"], format="%Y%m"
-        ).dt.date,
-    }
-
-    if not manual_update:
-        dynamic_assignment["version"] = 1
-
     # Finalize to write to sale.flag table
-    df = df[cols_to_write].assign(**dynamic_assignment).reset_index(drop=True)
```

Comment:
This code was actually redundant. It was assigning versions of 1 or the absence of a version column depending on the `manual_update` boolean. Either way, the existing data got incremented by 1 anyway if we were doing a manual update.